Samples/tutorials/excel-tutorial/README.md

```diff
@@ -0,0 +1,34 @@
+# Excel Tutorial - Completed
+
+This sample is the result of completing the [Tutorial: Create an Excel task pane add-in](https://learn.microsoft.com/office/dev/add-ins/tutorials/excel-tutorial). It was constructed with the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
+
+The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. This sample is best used if you've completed the tutorial and want to start from a more complete project. It's also helpful when debugging problems you encounter during the tutorial.
+
+## Run the add-in
+
+1. If the local web server is already running and your add-in is already loaded in Excel, proceed to step 2. Otherwise, start the local web server and sideload your add-in:
```

Comment:
I'm not following how you could already have the web server running. I think we should drop this step.